integration_test/third_party_apps_data/applications/nvml/centos_rhel/install

```diff
@@ -32,16 +32,24 @@ case $DEVICE_CODE in
         ;;
     *)
         # Installing latest version of NVIDIA CUDA and driver
-        DRIVER_VERSION=535.104.05
+        # Data Center/Tesla drivers and CUDA are released on different schedules;
+        # normally we install the matching versions of driver and CUDA 
+        # ($DRIVER_VERSION == $CUDA_BUNDLED_DRIVER_VERSION); due to https://github.com/NVIDIA/open-gpu-kernel-modules/issues/550
+        # we install a newer version of the driver
+        DRIVER_VERSION=535.129.03
         CUDA_VERSION=12.2.2
+        CUDA_BUNDLED_DRIVER_VERSION=535.104.05
         ;;
 esac
 
-echo "Installing NVIDIA CUDA $CUDA_VERSION with driver $DRIVER_VERSION"
+echo "Installing NVIDIA Data Center driver $DRIVER_VERSION"
 curl -fSsl -O https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
 sudo bash ./NVIDIA-Linux-x86_64-$DRIVER_VERSION.run --silent
-wget --no-verbose https://developer.download.nvidia.com/compute/cuda/$CUDA_VERSION/local_installers/cuda_${CUDA_VERSION}_${DRIVER_VERSION}_linux.run
-sudo sh cuda_${CUDA_VERSION}_${DRIVER_VERSION}_linux.run --toolkit --silent
+# Install the CUDA toolkit only, so that the CUDA toolkit uses the Data Center driver installed in the previous step
+# See https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html for CUDA and driver compatibility 
```

Comment:
[Optional] You should be able to omit the `index.html` suffix, as it's implied.
```suggestion
# See https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/ for CUDA and driver compatibility 
```

Reply:
Done!